content/docs/command-reference/dag.md

```diff
@@ -7,43 +7,25 @@ specified stage. By default it lists
 ## Synopsis
 
 ```usage
-usage: dvc pipeline show [-h] [-q | -v] [-c | -o] [-l] [--ascii]
-                         [--dot] [--tree]
-                         [targets [targets ...]]
+usage: dvc dag [-h] [-q | -v] [--dot] [--full] [target]
 
 positional arguments:
-  targets         DVC-files to show pipeline for. Optional.
-                  (Finds all DVC-files in the workspace by default.)
+  targets         Stage or output to show pipeline for.
+                  Optional. (Finds all stages in the workspace by default.)
 ```
 
 ## Description
 
-`dvc show` displays the stages of a pipeline up to one or more target DVC-files
-(stage files). If specific `targets` are omitted, `Dvcfile` will be assumed. The
-`-c` and `-o` options allow to list the corresponding commands or data file flow
-instead of stages.
-
-> Note that the stages in these lists are in descending order, that is, from
-> first to last.
+`dvc dag` displays the stages of a pipeline up to the target stage. If `target`
+is omitted, it will show the full project DAG.
 
 ## Options
 
-- `-c`, `--commands` - show pipeline as a list (diagram if `--ascii` or `--dot`
-  is used) of commands instead of paths to DVC-files.
-
-- `-o`, `--outs` - show pipeline as a list (diagram if `--ascii` or `--dot` is
-  used) of stage outputs instead of paths to DVC-files.
-
-- `--ascii` - visualize pipeline. It will print a graph (ASCII) instead of a
-  list of path to DVC-files. (`less` pager may be used, see
-  [Paging the output](#paging-the-output) below for details).
-
-- `--dot` - show contents of `.dot` files with a DVC pipeline graph. It can be
-  passed to third party visualization utilities.
-
-- `--tree` - list dependencies tree like recursive directory listing.
+- `--dot` - show DAG in `DOT` format. It can be passed to third party
+  visualization utilities.
 
-- `-l`, `--locked` - print locked stages only. See `dvc lock`.
+- `--full` - show full DAG that the target belongs too, instead of show DAG
+  consisting only of ancestors.
```

Comment:
```suggestion
- `--full` - show full DAG that the `target` belongs too, instead of showing
  the part that consists only of the target ancestors.
```